data.sql

```diff
@@ -38,3 +38,112 @@ update animals set owner_id=4 where id = 8 or id = 9 or id = 10;
 
 update animals set owner_id=5 where id = 5 or id = 6;
 
+-- inserting data in to the vet table
+
+insert into vets(name, age, date_of_graduction) 
+    VALUES ('William Tatcher', 45, '2000-04-23'),
+        (' Maisy Smith', 26, '2019-01-17'),
+        ('Stephanie Mendez', 64, '1981-05-04'),
+        ('Jack Harkness', 38, '2008-06-08');
+
+-- inserting data in to specializtions table
+
+insert into specializations(vets_id, species_id) 
+    VALUES (1,1),
+        (3,1),
+        (3,2),
+        (4,2);
+
+INSERT INTO visits (animals_id, vets_id, date_of_visit) 
+    VALUES
+    (
+        (SELECT id FROM animals WHERE name = 'Gabumon'), 
+        (SELECT id FROM vets WHERE name = 'Vet Jack Harkness'), 
```

Comment:
- [ ] There is no vet named `Vet Jack Harkness` but `Jack Harkness`. Please update all queries that have `Vet` in front of the vet's name because there are no existing records with such name. Take a look at lines 44 and 48 to see how you save the records.

Reply:
thanks for the feedback